stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_list_view.xml

```diff
@@ -12,6 +12,8 @@
         android:id="@+id/chatMessagesRV"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:clipToPadding="false"
+        android:paddingBottom="@dimen/stream_ui_spacing_small"
```

Comment:
Thank you 🙏  One more thing I was gonna add sometime, but never ended up choosing a PR for it 😄